srp/views.py

```diff
@@ -185,13 +187,25 @@ def srp_request_view(request, fleet_srp):
             character = EveManager.get_character_by_id(authinfo.main_char_id)
             srp_fleet_main = SrpFleetMain.objects.get(fleet_srp_code=fleet_srp)
 
+
             srp_request = SrpUserRequest()
             srp_request.killboard_link = form.cleaned_data['killboard_link']
             srp_request.additional_info = form.cleaned_data['additional_info']
             srp_request.character = character
             srp_request.srp_fleet_main = srp_fleet_main
-            srp_request.save()
 
+            try:
+                srp_kill_link = srpManager.get_kill_id(srp_request.killboard_link)
+                (srp_kill_data, ship_value) = srpManager.get_kill_data(srp_kill_link)
+            except:
```

Comment:
could be more intelligent and only catch `ValueError` exceptions (the ones your manager throws) as other errors do not necessarily mean the kill ID is invalid